src/lib/consensus/global_sub_window.ml

```diff
@@ -22,7 +26,7 @@ let sub a b = UInt32.sub a b
 let constant a = a
 
 module Checked = struct
-  module T = Mina_numbers.Global_slot
+  module T = Mina_numbers.Global_slot_since_genesis
```

Comment:
This shouid be `Global_slot_since_hard_fork.t`

Reply:
Going to use `Nat.Make32 ()`.

`of_global_slot` is using the `Global_slot_since_hard_fork.Checked.t` value in `Global_slot.t`